cmd/mountlib/mount.go

```diff
@@ -22,6 +25,30 @@ var (
 	ExtraFlags         *[]string
 )
 
+// Check is folder is empty
+func checkMountEmpty(mountpoint string) error {
+	fp, fpErr := os.Open(mountpoint)
+	defer fs.CheckClose(fp, &fpErr)
+
+	if fpErr != nil {
+		return errors.Wrap(fpErr, "Can not open: " + mountpoint)
```

Comment:
This line has styling error:
`-		return errors.Wrap(fpErr, "Can not open: " + mountpoint)`
`+		return errors.Wrap(fpErr, "Can not open: "+mountpoint)`

but I think it is false positive error.